From 546c7b5f590f17bb0039e7599b79e9f18ac9cf07 Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Fri, 29 Jul 2022 16:05:15 +0800
Subject: [PATCH] Reland "[WATCHLISTS] Add riscv watch"

This is a reland of commit 776b9eb9d9fce486ff3b8e542091649e66a29bc3

Original change's description:
> [WATCHLISTS] Add riscv watch
>
> Change-Id: I6e4dc69d6f22d3108ae74552b72bcafc0be3db64
> Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3793476
> Commit-Queue: Yahan Lu <yahan@iscas.ac.cn>
> Reviewed-by: Victor Gomes <victorgomes@chromium.org>
> Cr-Commit-Position: refs/heads/main@{#82092}

Change-Id: I33ff1808de80f03fdfd7c977b29a895c8110bae0
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3835293
Reviewed-by: Victor Gomes <victorgomes@chromium.org>
Commit-Queue: Yahan Lu <yahan@iscas.ac.cn>
Cr-Commit-Position: refs/heads/main@{#82577}
---
 WATCHLISTS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WATCHLISTS b/WATCHLISTS
index 6d82fdc0cbf..7be659ce458 100644
--- a/WATCHLISTS
+++ b/WATCHLISTS
@@ -130,6 +130,7 @@
     'arm': [
       'v8-mips-ports@googlegroups.com',
       'v8-ppc-ports@googlegroups.com',
+      'v8-risc-v-ports@chromium.org',
     ],
     'merges': [
       # Only enabled on branches created with tools/release/create_release.py
-- 
2.35.1

